### PR TITLE
ui: Search/filtering 'Filtered by:' search status

### PR DESCRIPTION
**Pull request by johncowen**
This PR adds a 'status' for the filtering/searching in the UI, without this its not super clear that you are filtering a recordset due to the menu selections being hidden once closed:

filter-by

This is the first area where we begin to use the i18n `t` helper, in this case to translate machine to human (en-US), which from a code perspective is probably the biggest change here. We've tried to get a good balance between providing some sort of nice fallback for non-translated text and sharing as many terms, words and phrases as possible.

Notes:

1. We cleaned up a tonne of variable names where we had confusing "should this name be singular or plural' patterns, most things here should now use singular naming patterns.
2. We reworked part of out PopoverSelect to convert OptGroup and Option to use glimmer and simplify things here, we also added a `required` property to specify that the menu should have at least 1 option selected.
3. Translation keys use `t` with its `default` fallbacks, if no key is found it will dash-split and titlecase the final part of the key so something like `common.brand.unknown-brand` will end up as `Unknown brand` if the translation doesn't exist.

This PR only contains this feature for the main Service listing, but will now be rolled out across the rest of the UI.

Internationalisation of text will be rolled out gradually as and when until we have a fully i18n'able UI, but moving forwards any new features should make use of `ember-intl` from the start.

Edit:

I decided to make this PR the base branch for integrating across the rest of the ui:
- [x] Top-level Service Listing (this PR)
- [x] Top-level, non-restricted listings (Nodes, KVs, Intentions) https://github.com/hashicorp/consul/pull/9497
- [x] Top-level, restricted listings (Namespaces, ACLs, Tokens, Policies, Roles) #9531
- [x] Improved testing/docs https://github.com/hashicorp/consul/pull/9608 
- [x] Secondary-level listings within Services (Upstreams, Linked Services, Instance Upstreams) https://github.com/hashicorp/consul/pull/9632
- [x] Secondary-level listings within Nodes (Health Checks, Instances) https://github.com/hashicorp/consul/pull/9622

### CODE REVIEW

File: ui/packages/consul-ui/translations/en-us.yaml [hunk 1]
```diff
@@ -1 +1,48 @@
 common:
+  brand:
```
Comment (kaxcode) [inline ui/packages/consul-ui/translations/en-us.yaml:2]: this is awesome!

**Review by kaxcode — APPROVED** [review]
LGTM

**Comment by johncowen** [comment]
Thanks for the review! I decided to make this PR the base branch for integrating across the rest of the ui, so I'll merge some other PRs down onto this before merging it all together onto master, so I added some TODO in the PR description.

File: ui/packages/consul-ui/translations/en-us.yaml (hunk 1 above)
Comment (kaxcode) [reply ui/packages/consul-ui/translations/en-us.yaml:2]: I can't remember if we discussed this. I do need to use a translation file atm for auth methods. Let's chat and get on the same page about this.

Comment (johncowen) [reply ui/packages/consul-ui/translations/en-us.yaml:2]: Sounds good, ping me when you are around and we can decide how we want to do it

**Pull request merged**